### PR TITLE
Add command to print public key of a keyfile

### PR DESCRIPTION
**Pull request by phy-chain**
For easy debugging, you can print the public key associated with a specific keyfile (useful for devnet especially to figure out which key you registered in case you generated several)

### CODE REVIEW

File: crates/cli/src/main.rs [hunk 1]
```diff
@@ -192,6 +194,13 @@ async fn main() {
             ),
             Err(err) => println!("Error during key file creation:{err}"),
         },
+        ConfCommands::PrintPublicKey => match gevulot_cli::keyfile::read_key_file(&args.keyfile) {
```
Comment (tuommaki) [inline crates/cli/src/main.rs:197]: `read_key_file()` returns `SecretKey`.

**Review by tuommaki — CHANGES_REQUESTED** [review]
Current implementation would print the secret key.

Please use `PublicKey::from_secret_key()` to extract public key.

File: crates/cli/src/main.rs (hunk 1 above)
Comment (phy-chain) [reply crates/cli/src/main.rs:197]: Oh yeah, you're right, misread the code !
Should be fixed.

**Comment by tuommaki** [comment]
Please address the `cargo fmt` errors :slightly_smiling_face:

**Comment by tuommaki** [comment]
To get this through, I went and implemented this once and for all in #172. Closing here.

**Pull request closed without merging**